pyproject.toml

```diff
@@ -24,39 +24,38 @@ packages = [
 cli = 'solvis.scripts.cli:cli'
 
 [tool.poetry.dependencies]
-boto3 = {version = "^1.26.82", optional = true}
 click = {version = "^8.1.3", optional = true}
 click-plugins = {version = "^1.1.1", optional = true}
 nshm-toshi-client = {version = "^1.0.0", optional = true}
 pyproj = "^3.3"
-python = ">=3.9,<4.0"
+python = ">3.10,<4.0"
```

Comment:
should it be `>=3.10` instead of `>3,10`?

Reply:
yep - good catch @voj thanks